packages/material/src/layouts/MaterialArrayLayout.tsx

```diff
@@ -148,25 +162,58 @@ class ExpandPanelRenderer extends React.Component<ExpandPanelProps, any> {
       <ExpansionPanel expanded={expanded} onChange={handleExpansion(childPath)}>
         <ExpansionPanelSummary expandIcon={<ExpandMoreIcon />}>
           <Grid container alignItems={'center'}>
-            <Grid item xs={11}>
+            <Grid item xs={10}>
               <Grid container alignItems={'center'}>
                 <Grid item xs={1}>
-                  <Avatar aria-label='Index'>{index + 1}</Avatar>
+                  <Avatar aria-label="Index">{index + 1}</Avatar>
                 </Grid>
                 <Grid item xs={2}>
                   {childLabel}
                 </Grid>
               </Grid>
             </Grid>
-            <Grid item xs={1}>
+            <Grid item xs={2}>
               <Grid container justify={'flex-end'}>
                 <Grid item>
-                  <IconButton
-                    onClick={removeItems(path, [index])}
-                    style={iconStyle}
+                  <Grid
+                    container
+                    direction="row"
+                    justify="center"
+                    alignItems="center"
                   >
-                    <DeleteIcon />
-                  </IconButton>
+                    {uischema.options && uischema.options.showSortButtons ? (
+                      <Fragment>
+                        <Grid item>
+                          <IconButton
+                            onClick={moveUp(path, index)}
+                            style={iconStyle}
+                            disabled={index == 0}
+                          >
+                            <ArrowUpward />
+                          </IconButton>
+                        </Grid>
+                        <Grid item>
+                          <IconButton
+                            onClick={moveDown(path, index)}
+                            style={iconStyle}
+                            disabled={isLast}
```

Comment:
I would suggest to unify the disable logic of Up/Down like in the MaterialLabelControl